CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ if (WIN32)
   option(HB_HAVE_DIRECTWRITE "Enable DirectWrite shaper backend on Windows" OFF)
 endif ()
 option(HB_BUILD_UTILS "Build harfbuzz utils, needs cairo, freetype, and glib properly be installed" OFF)
+option(HB_BUILD_TESTS "Build harbuzz tests" ON)
```

Comment:
"harfbuzz" :) Also feel free to apply naming more relevant to your internal use :)